pkg/reconciler/v1alpha1/taskrun/resources/output_resource.go

```diff
@@ -94,18 +85,12 @@ func AddOutputResources(
 			resourceContainers []corev1.Container
 			resourceVolumes    []corev1.Volume
 		)
-		// if resource is declared in input then copy outputs to pvc
```

Comment:
Nice. 
It's a NIT, but also important for consistency, I think the comment in https://github.com/tektoncd/pipeline/blob/33bddf9e2d873b2d8da0d27dfbb9961bbb35c8d6/pkg/reconciler/v1alpha1/taskrun/resources/output_resource.go#L50-L52 needs to be fixed.

Reply:
Thanks!